cli/flags.rs

```diff
@@ -13,7 +13,12 @@ macro_rules! svec {
 
 macro_rules! std_url {
   ($x:expr) => {
-    concat!("https://deno.land/std@v0.23.0/", $x)
+    concat!(
+      "https://deno.land/std@v",
+      env!("CARGO_PKG_VERSION"),
```

Comment:
```diff
macro_rules! std_url {
  ($x:expr) => {
    concat!(
      "https://deno.land/std@v",
-      crate::version::DENO
+      env!("CARGO_PKG_VERSION"),
      "/",
      $x
    )
  };
}
```

Theoretically, the value of `crate::version::DENO` should be read here.

But I'm not familiar with macro_rules.

`concat!` only accepts literals 😥

Reply:
@axetroy Try using `lazy_static!` instead - here's an example
https://github.com/denoland/deno/blob/a537c03df71da49c356bb56882be920e38d37558/cli/colors.rs#L12-L21

Reply:
This seems to be the simplest way of writing, there is no need to write with lazy_static.

`CARGO_PKG_VERSION` is the environment variable for cargo and is unlikely to change